### DISCUSSION TITLE
Show real file timestamp (creation date/modification date) in YYYY-MM-DD hh:mm:ss format

### DISCUSSION THREAD

**Discussion by OoZe1911** [filebrowser/filebrowser#1099] [Ideas]
Is it possible to replace the current file date shown in filebrowser (relative date from today) with the exact real creation date (and maybe add the modification date) in a classical format : YYYY-MM-DD hh:mm:ss

This is very hard to know when a file has been created or uploaded : 2 months ago is less relevant than the exact date for instance.
In other words in its current version this is impossible in filebrowser to know when a file has been created (date including hour/minutes/seconds).

Thanks a lot !

**Comment by playcase** [comment]
The same problem is that the date of the uploaded file is not displayed, and the sorting of the date format is also very important. I look forward to improvement.
